src/plotting/plotButtons.component.tsx

```diff
@@ -4,26 +4,23 @@ import { PlotDataset } from '../app.types';
 import { formatTooltipLabel } from './plot.component';
```

Comment:
Safe to say this function belongs in this component from now on, since this is the only place it's being used?

Reply:
True, I'll move it here